app/models/time_record.rb

```diff
@@ -26,4 +27,12 @@ def only_todays_task_could_be_activated
       errors.add(:time_start, I18n.t("time_records.errors.only_todays_taks"))
     end
   end
+
+  def value_of_spent_time
+    days_tasks = TimeRecord.where(assigned_date: assigned_date)
+    days_tasks = days_tasks.where.not(id: id) if id
+    if (days_tasks.sum(:spent_time) + spent_time) > 24.0
```

Comment:
What is really will happen? 
For example i started some task and sum of all it's day tasks is higher than 24 h. Maybe it because I forgot about it. Next day I open app and trying to start new task but previous one can not be stopped? Or is it stopped? If so what value should it have? 
What will happen if i close app with that state for three days? We'll have invalid time record with time_start <> nil. This record will be in .active scope. But we can't have active task in the past. #stop_other_launched_time_records method will be called which calls #stop which calls AR update! Update will not be able to save  invalid record and will raise an exception. 
It's kinda messy for me. 

I think app should have special action for this. If frontend tries to save record and sum of this record's day tasks is higher that 24h, this task should be forced to stop with maximum available value and user should get notification about it.